almanac.lic

```diff
@@ -14,6 +14,9 @@ class Almanac
     @almanac_skills = settings.almanac_skills
     @almanac = settings.almanac_noun
 
+    @startup_delay = settings.almanac_startup_delay || 15
```

Comment:
The || isn't necessary since this is in base.yaml

Reply:
Fixed. Thanks again for your reviews!